addons/hitreactions/XEH_postInit.sqf

```diff
@@ -0,0 +1,65 @@
+#include "script_component.hpp"
+
+GVAR(armComponents) = [
+    "leftshoulder",
+    "rightshoulder",
+    "lefthand",
+    "leftforearm",
+    "leftarmroll",
+    "rightforearm",
+    "rightarmroll",
+    "righthand",
+    "rightarm"
+];
+
+GVAR(safePickupDistance) = DEFAULT_PICKUP_DISTANCE;
+if (isNil QGVAR(undroppableGuns)) then {GVAR(undroppableGuns) = DEFAULT_UNDROPPABLE_GUNS};
+
+["ace_firedNonPlayer", {
```

Comment:
we should conditionally add these event handlers based on settings

Reply:
Done